StudCity.Frontend/src/App.js

```diff
@@ -7,6 +7,7 @@ import {
 import Main from "./pages/Main";
 import Authenticate from "./pages/Authenticate";
 import Registration from "./pages/Registration";
+import NotFound from "./pages/NotFound/NotFound";
```

Comment:
rename NotFound.js file to index.jsx instead so that naming would be the same as for lines 8-9